pkg/master/master.go

```diff
@@ -87,10 +87,10 @@ const (
 
 // Config is a structure used to configure a Master.
 type Config struct {
-	EtcdHelper    tools.EtcdHelper
-	EventTTL      time.Duration
-	MinionRegexp  string
-	KubeletClient client.KubeletClient
+	DataBaseStorage tools.StorageInterface
```

Comment:
Ah, that makes sense. We should rename the rest storage thingy, it has very
little to do with storage.

On Mon, Jul 27, 2015 at 12:06 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/master/master.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11806#discussion_r35509823>
> :
>
> > @@ -87,10 +87,10 @@ const (
> >
> >  // Config is a structure used to configure a Master.
> >  type Config struct {
> > -	EtcdHelper    tools.EtcdHelper
> > -	EventTTL      time.Duration
> > -	MinionRegexp  string
> > -	KubeletClient client.KubeletClient
> > +	DataBaseStorage tools.StorageInterface
>
> I didn't want to have "Storage", because we already have:
> "storage map[string]rest.Storage"
> and having "Storage" and "storage" is really misleading in my opinion.
>
> Changing to DatabaseStorage - I'm happy to change it in the future if you
> suggest better name for it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11806/files#r35509823>
> .
>